### PR TITLE
#697246 add call to action (CTA) support

### PR DESCRIPTION
**Pull request by vinisi**
**Motivation**
In order to support call-to-action (CTA) messages we created this component.

**Changes**
add CTA button
add call-to-action component
render call-to-action (CTA) message on cta_url type

**Reference**
https://developers.facebook.com/docs/whatsapp/cloud-api/messages/interactive-cta-url-messages

### CODE REVIEW

File: src/components/ApplicationJSon.vue [hunk 1]
```diff
@@ -65,11 +80,8 @@
       :unsupported-content-msg="translations.unsupportedContent"
       :position="position"
       :document="document"
-      :on-save="saveCard"
```
Comment (Plezito) [inline src/components/ApplicationJSon.vue:68]: Hello, why did you remove these parameters from the <menu-list-prompt>? Aren't they beeing used?

File: src/components/ApplicationJSon/CallToAction.vue [hunk 2]
```diff
@@ -0,0 +1,337 @@
+<template>
+  <div v-if="!isEditing" class="blip-container menu-list-prompt">
```
Comment (Plezito) [inline src/components/ApplicationJSon/CallToAction.vue:2]: Shoun't this class be called call-to-action instead of menu-list-prompt?

File: src/components/ApplicationJSon/CallToAction.vue [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+    </form>
+  </div>
+</template>
+
+<script>
+import { default as base } from '../../mixins/baseComponent.js'
+import { isFailedMessage } from '../../utils/misc'
+
+export default {
+  name: 'menu-list-prompt',
```
Comment (Plezito) [inline src/components/ApplicationJSon/CallToAction.vue:160]: Shouldn't the name of this component be 'call-to-action'?

**Review by Plezito — APPROVED** [review]

**Review by AndreSantosTake — APPROVED** [review]

File: src/components/ApplicationJSon/CallToAction.vue [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+  margin: 0px -10px;
+}
+
+.fixed-options ul .align-center {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  > bds-icon {
+    margin-right: 10px;
+    color: var(--color-primary, #1E6BF1) !important;
```
Comment (VMarchena) [inline src/components/ApplicationJSon/CallToAction.vue:347]:
```suggestion
    color: $color-primary !important;    
```
Why do not use the color from System design?

File: src/components/ApplicationJSon/CallToAction.vue [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+.fixed-options ul .align-center {
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
+  > bds-icon {
+    margin-right: 10px;
+    color: var(--color-primary, #1E6BF1) !important;    
+  }
+  > bds-typo {
+    color: var(--color-primary, #1E6BF1);
```
Comment (VMarchena) [inline src/components/ApplicationJSon/CallToAction.vue:350]:
```suggestion
    color: $color-primary;
```
Why do not use the color from System design?

**Review by VMarchena — APPROVED** [review]
